### DISCUSSION TITLE
basic_concurrency_test有极小的概率这样报错，请问可能的原因是什么

### DISCUSSION THREAD

**Discussion by lhyyy1** [SUSTech-CS305-Fall22/CS305-Project-Skeleton#80] [Q&A]
有如下两种类型：
第一种：
![image](https://user-images.githubusercontent.com/120730404/212084205-f44290df-ace7-4a0a-91af-47dda14821da.png)
第二种：
![image](https://user-images.githubusercontent.com/120730404/212084237-49a2bad1-98e0-49e9-b6a0-d1645dbc04c4.png)

**Comment by TANSixu** [comment]
Hi, 应该是读了一个空文件。可能写入文件那里出了一点问题。
